src/Components/BottomTabs.jsx

```diff
@@ -1,14 +1,26 @@
 import {useLocation, useNavigate} from 'react-router-dom';
 import {Cog8ToothIcon, HomeIcon, QrCodeIcon} from '@heroicons/react/20/solid';
+import useAppState from '../hooks/useAppState';
+import {useIsOffline} from '../utils/client';
 
 const BottomTabs = () => {
   const navigate = useNavigate();
   const {pathname} = useLocation();
+  const offline = useIsOffline();
+  const {enableModal} = useAppState();
 
-  if (pathname === '/scan') {
+  if (pathname === '/scan' || pathname === '/auth/redirect') {
     return null;
   }
 
+  const goToScan = () => {
+    if (offline) {
+      enableModal('You are offline', 'Scanning QR codes requires an internet connection');
```

Comment:
is this for the event qr code or the participant qr code? at least being able to show someone's details from their ticket qr code should be possible while offline as long as we already have the data locally

Reply:
Good point - currently I don't check if we already have the person or not, but if the person already exists locally we can show it